features/container/sentinel/src/main/filtered-resources/etc/org.apache.karaf.features.cfg

```diff
@@ -33,7 +33,7 @@ featuresRepositories = \
 #
 featuresBoot = \
     (aries-blueprint, \
-    deployer), \
+    deployer, \
```

Comment:
Think we shouldn't need to change any of this grouping.

Reply:
I think this is only necessary because the wrap feature needs to be installed before the custom features, that's why I added earlier wrap under owsap-encoder feature.
Otherwise I start getting an exception when running runInPlace.sh 
java.lang.IllegalStateException: Unknown protocol: wrap
at
org.apache.felix.framework.URLHandlersStreamHandlerProxy.parseURL

Reply:
okay, let me reproduce locally.